### DISCUSSION TITLE
某一分区删除部分数据后，select无结果，但是show partitions显示有数据

### DISCUSSION THREAD

**Discussion by linlin994395** [apache/doris#20821] [A - General / Q&A]
有一个表，对于其中一个分区，存储了多个日志，删除其中部分日志，然后过几天后查询发现，整个分区无数据。
![image](https://github.com/apache/doris/assets/22185640/30d60cdc-ef08-4ebd-8c0e-118a7d3bf7e9)
![image](https://github.com/apache/doris/assets/22185640/257310ee-1a28-4702-aeab-a6d382847ce4)

**Comment by yagagagaga** [comment]
What's your Doris version and how can I reproduce?

**Reply by linlin994395** [reply]
Doris version: doris-1.2.2-rc01
reproduce steps:
steps 1:  create dynamic table, "dynamic_partition.time_unit" = "DAY"
steps 2: flink write to this table from kafka topic, which contains multi event_name such as LogLogin LogPay LogReg LogOnline,,
steps 3: some days later, i found storage is huge, and some event_name is unuseful, so i delete those event_name 
delete from table where pt_dt between '2023-01-01' and '2023-01-02' and event_name in ('LogOnline')
steps 4: SHOW DATA FROM   SHOW PARTITIONS FROM shows storage has been reduced, but i found in some partition , it does not have any data. but SHOW PARTITIONS FROM table where PartitionName = ('p20230508' ) shows it has data

**Comment by Hby118** [comment]
现在呢,过了一天看是不是还是显示有数据

他这个好像都是另起一个线程去处理,不是马上出结果

**Reply by linlin994395** [reply]
已经尝试好几天了，select还是没有结果。
SHOW PARTITIONS显示有数据
![image](https://github.com/apache/doris/assets/22185640/bd118215-f4e5-44cf-b4f8-5a6288c33868)
SHOW TABLETS FROM table PARTITIONS(p20230508); 显示也有数据
![image](https://github.com/apache/doris/assets/22185640/aeb3b2fa-7c6d-4945-87cc-0f13022bab08)
SHOW TABLET 209191; 也没问题
![image](https://github.com/apache/doris/assets/22185640/6013a071-f3bb-4665-9ad7-21946f436e64)
SHOW PROC '/dbs/10007/27438/partitions/209190/27439/209191'; 也没问题
![image](https://github.com/apache/doris/assets/22185640/015ec29e-da2e-45ff-8eda-3a386dea56e5)

就是select查不出来数据，还没定位到原因

**Comment by linlin994395** [comment]
今天select 有数据了，但是反复执行如下sql，显示信息不一致。这是为什么？和元数据有关？
![image](https://github.com/apache/doris/assets/22185640/421a4113-8b36-48c1-a9dc-ca0f19df14d7)
![image](https://github.com/apache/doris/assets/22185640/f09c3356-df50-4081-b7e9-997160c4dcca)
![image](https://github.com/apache/doris/assets/22185640/e975b46c-653a-46f0-8133-3338f2cf6505)
